core/src/main/java/com/mapzen/android/graphics/OverlayManager.java

```diff
@@ -814,4 +871,13 @@ private MapData addPointToMarkerMapData(Marker marker) {
     findMe.setActivated(false);
     return false;
   }
+
+  @Override public boolean onRotate(float x, float y, float rotation) {
+    final View compass = mapView.findViewById(R.id.mz_compass);
```

Comment:
Sure, I did not realized this. I have basically looked at how are onFling() and onPan() methods written. So in these methods it will be better to replace mapView.findViewById(R.id.mz_find_me) with getFindMe() too.